utils/testutils.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+import "github.com/NethermindEth/juno/core/felt"
+
+func HexToFelt(hex string) *felt.Felt {
+	// We know our test hex values are valid, so we'll ignore the potential error
+	f, _ := new(felt.Felt).SetString(hex)
+	return f
+}
```

Comment:
I am not sure that this should be in the utils package. It will encourage people to just use this over creating a new felt and will not be limited to tests.  This can lead to us just ignoring errors. I think it is ok to have a private function in each package where required as a private function. You can consider declaring the function as a variable in the test function. That way no one in the rest of the package will have access to that but the test function where it is needed.

This will open doors to ignoring errors.

Reply:
Done, removed the public function HexToFelt